lib/eventFactory.js

```diff
@@ -471,20 +473,31 @@ class EventFactory extends BaseFactory {
                 return p;
             })
             .then(pipeline => pipeline.token
-                .then(token => this.scm.decorateAuthor({ // decorate user who creates this event
-                    username,
-                    scmContext,
-                    token
-                }).then((creator) => {
-                    modelConfig.creator = creator;
+                .then((token) => {
+                    if (!modelConfig.creator) {
```

Comment:
is there anything else in the event which marks it as a periodic build ? wondering if creator should be configurable or should be inferred from another field which stands for periodic build.

Reply:
just going to use `creator` and `cause message`
the executor-queue will pass this info in when starting an event